github/ci/prow/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -0,0 +1,69 @@
+periodics:
+- name: periodic-kubevirtci-sync-crio-repository-weekly
+  cron: "25 0 * * 1"
+  decorate: true
+  annotations:
+      testgrid-create-test-group: "false"
+  decoration_config:
+    timeout: 1h
+    grace_period: 5m
+  max_concurrency: 1
+  labels:
+    preset-shared-images: "true"
+  spec:
+    containers:
+      - image: kubevirtci/bootstrap:v20201119-a5880e0
+        env:
+          - name: GOOGLE_APPLICATION_CREDENTIALS
+            value: /etc/gcs/service-account.json
+          - name: OS
+            value: CentOS_8_Stream
+          - name: BUCKET_DIR
+            value: kubevirtci-crio-mirror
+          - name: LOCAL_MIRROR_DIR
+            value: opensuse-mirror
+          - name: BASE_REPOID
+            value: devel_kubic_libcontainers_stable
+          - name: CRIO_VERSIONS
+            value: "1.20"
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - |
+            set -e;
```

Comment:
The job looks great, just one general note for the future: I think it is preferrable to keep all jobs, no matter if golang, bash, or whatever defined in the normal source code and not inside the job definitions.

The pattern is adding `project-infra` as `extraRef` to the job and defining it as the "current directory". Then one can simply invoke the script here and fixing scripts is still just a change on the PR and the next job run picks up the new version.

That is better maintain- and testable.

Reply:
I agree, thanks. I'll refactor the job as you suggest